conf/layer.conf

```diff
@@ -9,6 +9,7 @@ BBFILE_COLLECTIONS += "freescale-layer"
 BBFILE_PATTERN_freescale-layer := "^${LAYERDIR}/"
 BBFILE_PRIORITY_freescale-layer = "5"
 LAYERSERIES_COMPAT_freescale-layer = "langdale mickledore"
+LAYERDEPENDS_freescale-layer = "core openembedded-layer"
```

Comment:
Hello! I was wondering if I could make a request regarding `linux-fslc`. Is it possible to take out the need for `lzop-native` and make `meta-oe` something that isn't required? If it's not too much trouble, could you create a PR for this? Thank you so much!

Reply:
Please check #1544.